index.php

```diff
@@ -140,12 +139,12 @@
             <div class="col-md-4">
                 <h2 class="section-header">Featured Video </h2>
                 <div class="embed-responsive embed-responsive-16by9 img-thumbnail">
-                    <iframe class="embed-responsive-item" src="https://www.youtube-nocookie.com/embed/37GBEBLfhWA?;wmode=transparent" allowfullscreen></iframe>
+                    <iframe class="embed-responsive-item" src="https://www.youtube-nocookie.com/embed/PFNH1x9r2uM?list=PLL1agQEsn8lTo09Px9e1RgcrX6TAwfIS7;wmode=transparent" allowfullscreen></iframe>
                 </div>
                 <br><br>
                 <p><a class="btn btn-default" href="https://www.youtube.com/user/TopGun1294/"><i class="fa fa-youtube-play"></i> Visit our Youtube Channel &raquo;</a></p>
                 <h2 class="section-header">Featured Picture</h2>
-                <img class="img-responsive img-thumbnail" alt="Our 2016 Robot" src="/img/featured_picture.jpg" />
```

Comment:
More information: It looks like you merged the changes from `master` in 64a05737dac10f3442eb329e6a9847749bcc047d, but then in 13c2be7b7bff6c703c07f65dafa673525ff47455 you had an old version of the file which overwrote it. You might be able to just `git reset`/`git revert` the specific file, but I'm not sure about the specific syntax.